src/doc/gui_manual/Contributing/index.rst

```diff
@@ -295,101 +301,52 @@ a lot of guidance on constructing math equations with LaTeX.
-
-We use a third party extension (e.g. not a builtin) to Sphinx for spell checking
-`sphinx-contrib.spelling <http://sphinxcontrib-spelling.readthedocs.io/en/latest/index.html>`_
-which requires `PyEnchant <https://pythonhosted.org/pyenchant/>`_ and adds
-support for a custom ``.. spelling::`` directive.
-
-If a spell check encounters any spelling errors, it will emit them along
-with the file name and approximate line number at which they occur. It will
-also output any spelling errors to a file, ``output.txt`` in the ``_spelling``
-build directory. The line numbers Sphinx reports for the spelling errors it
-encounters are not the input text file line numbers. They are close but rarely
-exactly the line numbers of the input text file. Its best to simply search the
-document for the flagged words.
-
-Correcting Flagged Words
-~~~~~~~~~~~~~~~~~~~~~~~~
-To correct a given spelling error, your options are...
-
-* Make a correction or other adjustment to the flagged word(s).
-* Add *special cases* to a ``.. spelling::`` directive at the end of the
-  ``.rst`` file.
-* Add *common* words, to the global ``spelling_wordlist.txt`` file.
-
-Much of the VisIt_ documentation includes the names of executable applications,
-their arguments, GUI widgets, VisIt_ components and VisIt_ architectural details
-and which are often not real words. It is best to typeset such names *exactly*
-as a user might encounter them while using VisIt_. But, adding such words to the
-global ``spelling_wordlist.txt`` makes sense only if the word is commonly used
-*throughout* VisIt_ documentation. Otherwise, it is best to treat it and other
-situations like it as a *special case* and add it *only* to a ``.. spelling::``
-directive at the end of the ``.rst`` file where it is used. For example, ``fmt``
-is a word used in describing :ref:`movie tools <Movie tools>` but not elsewhere
-in VisIt_. So, rather than add ``fmt`` to the global ``spelling_wordlist.txt``
-file, we add it at the end of :file:`../Animation/Movie_tools.rst` like so...
+Spell Checking Using Aspell
+---------------------------
+You can do a pretty good job of spell checking using the Unix/Linux ``aspell``
+command.
 
-.. code-block:: RST
+#. Run ``aspell`` looking for candidate miss-spelled words.
 
-    .. spelling::
-        fmt
+   .. code-block:: shell
 
-How Spell Check Works
-~~~~~~~~~~~~~~~~~~~~~
-The ``.. spelling::`` directive is a *custom* extension to Sphinx. It is not
-a builtin extension. This means that other documentation contributors wanting
-to make a local build of the documentation before committing their changes would
-be *required* to have the additional dependencies installed to support spell
-checking whether or not they ever needed to run a spell check.
+       find . -name '*.rst' -exec cat {} \; | \
+       grep -v '^ *.. image:\|figure:\|code:\|_' | \
+       tr '`' '@' | sed -e 's/\(@.*@\)//' | \
+       aspell --ignore-case -p ./aspell.en.pws list | \
```

Comment:
We might want to remove the flag to ignore case

Reply:
Sure. I can't recall why I thought I needed that but will do. Thx.